storage/artifactory.go

```diff
@@ -41,6 +44,8 @@ type ArtifactoryList struct {
 	Files []ArtifactoryFile `json:"files"`
 }
 
+var _ Storage = (*Artifactory)(nil)
```

Comment:
I swear I knew what this did at some point but having trouble remembering/searching.  What does this do again?

Reply:
It forces the struct type to implement the interface. So at compile time, you get a nice error if you are missing a method or w/e. Without this, the compile error appears wherever you first use it. Which is less obvious imo.

It also tells anyone looking at this file that it should implement this interface

Reply:
Ahhh ok awesome that is very handy indeed.  Thank you!